src/hotspot/share/runtime/os.hpp

```diff
@@ -900,22 +900,22 @@ class os: AllStatic {
   static int get_native_stack(address* stack, int size, int toSkip = 0);
 
   // General allocation (must be MT-safe)
-  static void* malloc  (size_t size, MEMFLAGS flags, const NativeCallStack& stack);
-  static void* malloc  (size_t size, MEMFLAGS flags);
-  static void* realloc (void *memblock, size_t size, MEMFLAGS flag, const NativeCallStack& stack);
-  static void* realloc (void *memblock, size_t size, MEMFLAGS flag);
+  static void* malloc  (size_t size, MemTag mem_tag, const NativeCallStack& stack);
+  static void* malloc  (size_t size, MemTag mem_tag);
+  static void* realloc (void *memblock, size_t size, MemTag mem_tag, const NativeCallStack& stack);
+  static void* realloc (void *memblock, size_t size, MemTag mem_tag);
 
   // handles null pointers
   static void  free    (void *memblock);
-  static char* strdup(const char *, MEMFLAGS flags = mtInternal);  // Like strdup
+  static char* strdup(const char *, MemTag mem_tag = mtInternal);  // Like strdup
   // Like strdup, but exit VM when strdup() returns null
-  static char* strdup_check_oom(const char*, MEMFLAGS flags = mtInternal);
+  static char* strdup_check_oom(const char*, MemTag mem_tag = mtInternal);
 
   // SocketInterface (ex HPI SocketInterface )
   static int socket_close(int fd);
-  static ssize_t recv(int fd, char* buf, size_t nBytes, uint flags);
-  static ssize_t send(int fd, char* buf, size_t nBytes, uint flags);
-  static ssize_t raw_send(int fd, char* buf, size_t nBytes, uint flags);
+  static ssize_t recv(int fd, char* buf, size_t nBytes, uint type);
+  static ssize_t send(int fd, char* buf, size_t nBytes, uint type);
+  static ssize_t raw_send(int fd, char* buf, size_t nBytes, uint type);
```

Comment:
This set of changes is wrong.  These aren't MEMFLAGS flags.
(I hope there aren't any more like this.  This sort of thing would be easy to miss in a change this
large.  If I were making this change I'd have broken it up into several smaller pieces.)

Reply:
Looks like a find/replace issue. Reverted the change.